src/Components/src/Microsoft.AspNetCore.Components/IComponent.cs

```diff
@@ -12,12 +14,12 @@ public interface IComponent
         /// Initializes the component.
         /// </summary>
         /// <param name="renderHandle">A <see cref="RenderHandle"/> that allows the component to be rendered.</param>
-        void Init(RenderHandle renderHandle);
+        void Configure(RenderHandle renderHandle);
 
         /// <summary>
         /// Sets parameters supplied by the component's parent in the render tree.
         /// </summary>
         /// <param name="parameters">The parameters.</param>
```

Comment:
Could we have an XML doc comment for the `returns` value? Something like (with syntax corrected):

`Returns a <see cref="Task"/> that completes when the component has finished updating and rendering itself.`

Reply:
Done